Plutarch/Api/V1/Scripts.hs

```diff
@@ -64,3 +65,5 @@ deriving via
   (DerivePConstantViaNewtype Plutus.ValidatorHash PValidatorHash PByteString)
   instance
     (PConstant Plutus.ValidatorHash)
+
+-----------------------------
```

Comment:
It was originally sectioning functions that have now been moved, I can remove rq (just to avoid the chance of it causing a conflict elsewhere somehow)

Reply:
I did not expect that to break the format check :upside_down_face: , wow